addon/ng2/utilities/get-dependent-files.ts

```diff
@@ -0,0 +1,109 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as ts from 'typescript';
+
+/**
+ * Declare a MapType where key is a string and value is an array of type string
+ * Use for storing a file's path as key and its dependent module's relative path
+ */
+declare type ImportMap {
+  [key: string]: ImportMap | string[];
+}
+
+/**
+ * Create a SourceFile as defined by Typescript Compiler API.
+ * Generate a AST structure from a source file.
+ * 
+ * @param fileName source file for which AST is to be extracted
+ */
+export function createtsSourceFile(fileName: string): ts.SourceFile {
+  return ts.createSourceFile (
```

Comment:
Styling issues: this is hard to read. No spaces between the function and `(`, put the first argument on the same line as the call and horizontally align additional arguments with it. Align the `)` at the end of the last argument.